test/parser/expression/AssociativeArrayLiterals.test.js

```diff
@@ -135,6 +135,17 @@ describe("parser associative array literals", () => {
         });
     });
 
+    it('allows separating properties with colons', ()=>{
+        const { tokens } = brs.lexer.Lexer.scan(`
```

Comment:
In cases where location doesn't matter, please use the `identifier` and `token` convenience functions (see the other tests in this file for examples) to keep these tests isolated from the lexer's output.

Reply:
Sure. I updated the test to use an array of tokens like you asked, so you should be good to